docs/source/quickstart.md

```diff
@@ -17,7 +17,7 @@ Before installing JupyterHub, you will need:
 
   - If you are using **`pip`**, install a recent version of
     [nodejs/npm](https://docs.npmjs.com/getting-started/installing-node).
-    For example, install it on Linux (Debian/Ubuntu) using:
+    For example, install it on Linux (Debian/Ubuntu) using the following:
```

Comment:
I don't think it's beneficial to add `the following` when a code block fits into the flow of a sentence, e.g. "install it...using: [code block]". Adding `following` is helpful when it's further removed from the code block, as in

> Running the following code will do such and such:

I know we've been incosistent with this, but I'd like to mostly move _away_ from habitual "with the following:"

Reply:
```suggestion
    For example, install it on Linux (Debian/Ubuntu) using:
```